### PR TITLE
docs: Minor Clarifications to README

### PR DESCRIPTION
**Pull request by YonLiud**
I have made some minor updates to the README file to improve clarity and readability.

Please review and let me know if further adjustments are needed :)

### CODE REVIEW

**Review by freakwriter — APPROVED** [review]
@jeanpaul looks good to me!

**Comment by jeanpaul** [comment]
Hi @YonLiud 
Thank you for your contribution to the Starter Kit! Your PR requires a few merge-conflicts to be resolved, can you fix these?

**Comment by YonLiud** [comment]
Sure!
I will get right into it in a few moments

**Comment by YonLiud** [comment]
I believe I’ve resolved the merge conflict in the `README.md`.
Please keep me posted about any other issues

File: README.md [hunk 1]
```diff
@@ -27,19 +25,28 @@ store with an comprehensive API
 ✅ [**PostgreSQL**](https://www.postgresql.org/) -  Workhorse of the Data
 Engineering world, handles large amounts of data safely.
 
-### What you can build
+### What You Can Build
```
Comment (freakwriter) [inline README.md:28]:
```suggestion
### What you can build
```

File: README.md [hunk 2]
```diff
@@ -116,9 +121,9 @@ language model and Qdrant as your vector store.
 
 ## Upgrading
 
-### For Nvidia GPU users
+* ### For Nvidia GPU Setups:
```
Comment (freakwriter) [inline README.md:124]:
```suggestion
* ### For Nvidia GPU setups:
```

File: README.md [hunk 3]
```diff
@@ -130,9 +135,9 @@ docker compose pull
 docker compose create && docker compose up
 ```
 
-### For everyone else
+* ### For Non-GPU Setups:
```
Comment (freakwriter) [inline README.md:138]:
```suggestion
* ### For Non-GPU setups:
```

**Review by freakwriter — CHANGES_REQUESTED** [review]
@YonLiud I added some suggested commits to shift the titles to sentence case, to match the rest of the file. Can you commit those suggestions and then we can merge? These are helpful changes. :)

**Comment by YonLiud** [comment]
```git
grep -v -e'^<<<<<<<' -e '^>>>>>>>' -e'=======' filename.txt > filename.txt.merged
```

**Review by freakwriter — APPROVED** [review]
@jeanpaul can you merge this?

**Review by jeanpaul — APPROVED** [review]

**Pull request merged**